spec/features/proposal/collection_draft_proposals/collection_draft_proposal_creation_spec.rb

```diff
@@ -76,4 +76,20 @@
       end
     end
   end
+
+  context 'when viewing a collection' do
```

Comment:
We will also want to create an update request from viewing a collection. Therefore, either we can add text to this line to say something like 'when viewing a collection to create a delete metadata proposal', _or_ we can add another context on/before line 87 for the delete request so the before block can be used later for update requests also.